tests/unit/test_text_splitters.py

```diff
@@ -0,0 +1,54 @@
+#  Neo4j Sweden AB [https://neo4j.com]
```

Comment:
The unit test should try and follow the same structure as the src/ folder

Reply:
Done!